app/src/main/java/org/piramalswasthya/sakhi/configuration/CDRFormDataset.kt

```
@@ -218,18 +292,66 @@ class CDRFormDataset(
+        }
+    }
+    private fun convertTimeToLong(value: Any?): Long {
+        return when (value) {
+            is Long -> value
+            is String -> {
+                try {
+                    val parts = value.split(":")
+                    val hours = parts.getOrNull(0)?.toIntOrNull() ?: return 0L
+                    val minutes = parts.getOrNull(1)?.toIntOrNull() ?: return 0L
+                    return (hours * 60 * 60 * 1000 + minutes * 60 * 1000).toLong()
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)